test/libsolidity/smtCheckerTests/types/array_literal_5.sol

```diff
@@ -0,0 +1,16 @@
+pragma experimental SMTChecker;
+
+contract C
+{
+    uint[] s;
+	function f() public {
```

Comment:
Indentation

Reply:
Fixed, but will need to update test expectation now

Reply:
Ok!